### DISCUSSION TITLE
I cannot access my omada container

### DISCUSSION THREAD

**Discussion by hasrt1206** [tteck/Proxmox#1308] [Help Needed]
Hi! I am unable to access my omada container using the ip and port given after using your script. What could the problem be?

**Comment by tteck** [comment]
New install?

**Reply by hasrt1206** [reply]
Yes. I have used the default port 8043 but I cannot connect to the web interface

**Reply by tteck** [reply]
Are you using https?

**Reply by hasrt1206** [reply]
Yes

**Reply by tteck** [reply]
Did the script finish?

**Comment by tteck** [comment]
No issue here:

```
   ____                      __     
  / __ \____ ___  ____ _____/ /___ _
 / / / / __  __ \/ __  / __  / __  /
/ /_/ / / / / / / /_/ / /_/ / /_/ / 
\____/_/ /_/ /_/\__,_/\__,_/\__,_/  
 
Using Advanced Settings
Using Ubuntu Version: 20.04
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 112
Using Hostname: omada
Using Disk Size: 8
Allocated Cores: 2
Allocated RAM: 2048
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Omada LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.253 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.113.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Installed Omada Controller
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Omada should be reachable by going to the following URL.
         https://192.168.86.253:8043 
```
![Screenshot 2023-04-08 7 28 05 PM](https://user-images.githubusercontent.com/21342670/230746728-055d4a55-69b1-4634-a4aa-578fe9580d7a.png)

**Reply by hasrt1206** [reply]
![Omada](https://user-images.githubusercontent.com/69708609/230746844-34cac536-d8fc-4ef5-9674-1dfefdb9edb7.gif)

I am able to ping my address but not connect to the web interface

**Reply by tteck** [reply]
https?

**Reply by hasrt1206** [reply]
![Omada 2](https://user-images.githubusercontent.com/69708609/230746958-017c36ff-077d-4d67-8e29-8100e53d835d.gif)
Yes, I am using https

**Reply by tteck** [reply]
My recommendation would be to delete the container and then execute the script again, ensuring that it runs to completion. As I demonstrated earlier, there is no problem with the script.

**Reply by hasrt1206** [reply]
I tried using a mobile device and it does connect but not on my desktop browser.

Thanks for all your help!
